src/shared/UserContext.ts

```diff
@@ -0,0 +1,22 @@
+import { createContext } from "react";
+
+export type User = {
```

Comment:
This is the user type, that we can access anywhere in the app, thanks to the provider in `App.tsx`